### PR TITLE
Use dedicated option tlvs

### PR DESCRIPTION
**Pull request by archerhungbsn**
Reviewer: @poolakiran @harishraghavendra

### CODE REVIEW

**Comment by bsn-abat** [comment]
Find artifact changes from this pull request at https://github.com/floodlight/loxigen-artifacts/tree/PR-691-b1

File: openflow_input/bsn_tlv [hunk 1]
```diff
@@ -1649,20 +1649,14 @@ struct of_bsn_tlv_no_drop : of_bsn_tlv {
     uint16_t length;
 };
 
-struct of_bsn_tlv_dhcpv6_option : of_bsn_tlv {
+struct of_bsn_tlv_dhcpv6_opt_remote_id : of_bsn_tlv {
```
Comment (poolakiran) [inline openflow_input/bsn_tlv:1652]: Can we just call it remote_id, so that we can repurpose it in the future.

File: openflow_input/bsn_tlv [hunk 2]
```diff
@@ -1649,20 +1649,14 @@ struct of_bsn_tlv_no_drop : of_bsn_tlv {
...
-
-enum ofp_bsn_dhcpv6_opt_fmt(wire_type=uint16_t) {
-    OFP_BSN_DHCPV6_OPT_FMT_OPAQUE_DATA = 1,
-    OFP_BSN_DHCPV6_OPT_FMT_AGENT_DATA = 2,
-    OFP_BSN_DHCPV6_OPT_FMT_PREFIX_OPAQUE_AND_AGENT_DATA = 3,
+    of_octets_t value;
 };
 
-struct of_bsn_tlv_dhcpv6_opt_fmt : of_bsn_tlv {
+struct of_bsn_tlv_dhcpv6_opt_client_ll_addr : of_bsn_tlv {
```
Comment (poolakiran) [inline openflow_input/bsn_tlv:1658]: Similarly, can  we call it as client_ll_addr?

**Review by poolakiran — COMMENTED** [review]
Couple of comments, LGTM otherwise.

**Comment by bsn-abat** [comment]
Find artifact changes from this pull request at https://github.com/floodlight/loxigen-artifacts/tree/PR-691-b2

**Pull request merged**